### PR TITLE
Initial Wings3D release

### PR DESCRIPTION
**Pull request by dgud**
I have got support requests of a flatpak/flathub Wings3d release.

So here is my initial try, don't really know what I'm doing so be helpful with the
stuff I got wrong.

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build com.wings3d.WINGS
testbot, build com.wings3d.WINGS

**Comment by nedrichards** [comment]
On the opencl thing you might want to talk directly to the [FDo SDK people](https://gitlab.com/freedesktop-sdk/freedesktop-sdk).

File: com.wings3d.WINGS.json [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+    "cleanup": ["/lib/erlang", "/bin/ct_run", "/bin/*er", "/bin/epmd",
+		"/bin/*erl*", "/bin/rebar3", "escript",	"/bin/wx*",
+		"/include"],
+
+    "modules": [
+	"shared-modules/glu/glu-9.0.0.json",
+	{
+            "name": "wxWidgets-311",
+            "buildsystem": "autotools",
```
Comment (nedrichards) [inline com.wings3d.WINGS.json]: this is default so not strictly required.

**Comment by nedrichards** [comment]
Overall, this would be much more readable if you ran it through a formatter (I tend to use jq but your mileage may vary). Other than that - it all looks fine. I'll test the build when I get a chance and see if there's anything else obvious.

File: com.wings3d.WINGS.json [hunk 2]
```diff
@@ -0,0 +1,130 @@
+{
+    "app-id": "com.wings3d.WINGS",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "wings",
+    "rename-icon": "wings",
```
[the remark this answers is not in the document]
Comment (dgud) [reply com.wings3d.WINGS.json]: The icons are already copied out, and not used.

**Comment by nedrichards** [comment]
You need to add this lot for dconf (sorry it's so ungainly, this is being fixed)
```
 "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
"--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
```
I didn't try installing a plugin but you will want to test that.

**Comment by dgud** [comment]
Improved formatting (hopefully).
Added donf config, removed autotools.

I had to read the code to see what the plugin stuff does, long time since implemented and not used,
and it's broken on all platforms and it have never worked if wings have been installed outside of the home directory :-)

So I think we can ignore the plugin issue for now, I need something to release in the coming releases.
I noticed that there is also a way to add patches to wings, which should work, wings read the app 'data_dir' but I have not tested that code in years either, so that it's probably broken as well.

**Comment by nedrichards** [comment]
bot, build com.wings3d.WINGS
testbot, build com.wings3d.WINGS

**Comment by nedrichards** [comment]
@dgud is there anyone else who needs access to the repo?

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.wings3d.WINGS

**Comment by dgud** [comment]
@nedrichards  You can add access to @bjorng  i.e. https://github.com/bjorng

**Pull request closed without merging**